pkg/services/rpcsrv/server.go

```diff
@@ -2449,6 +2457,10 @@ func getRelayResult(err error, hash util.Uint256) (any, *neorpc.Error) {
 		return result.RelayResult{
 			Hash: hash,
 		}, nil
+	case errors.Is(err, network.ErrSender):
```

Comment:
This error should have more meaningful name. As you can see, it's in the `network` package, thus, `network.ErrSender` can be almost about anything network-related. Use something like `network.ErrInvalidNotaryRequestSender` instead.

Reply:
Yes, fixed